robots/assembly/content_metadata_create.rb

```diff
@@ -0,0 +1,31 @@
+require_relative './base'
+
+module Robots
+  module DorRepo
+    module Assembly
+      class ContentMetadataCreate < Robots::DorRepo::Assembly::Base
+        def initialize(opts = {})
+          super('dor', 'assemblyWF', 'content-metadata-create', opts)
+        end
+
+        def perform(druid)
+          ai = item(druid)
+          return LyberCore::Robot::ReturnState.new(status: :skipped, note: 'object is not an item') unless ai.is_item? # not an item, skip
+
+          if ai.stub_content_metadata_exists? && ai.content_metadata_exists? # both stub and regular content metadata exist -- this is an ambiguous situation and generates an error
+            return LyberCore::Robot::ReturnState.new(status: :skipped, note: "#{Dor::Config.assembly.stub_cm_file_name} and #{Dor::Config.assembly.cm_file_name} both exist")
```

Comment:
yup, ok...except for the robot to send an error, we just raise it and it caught later (status: error is not an option currently)